### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by buzz-js**

### CODE REVIEW

**Comment by antfu** [comment]
Well, I wasn't really intending to make this project i18n complete, as the texts are just placeholders for demonstrating the functionality and would eventually be removed by the template users. But anyway, as long as you have already done the work, I am happy to merge it. Thanks.

And you might need to add an entry for `es` in `src/message.ts`.

**Comment by buzz-js** [comment]
> Well, I wasn't really intending to make this project i18n complete, as the texts are just placeholders for demonstrating the functionality and would eventually be removed by the template users. But anyway, as long as you have already done the work, I am happy to merge it. Thanks.
> 
> Before I do, you might need to add an entry for `es` in `src/message.ts` :)

I have really done it because I want to contribute my grain of sand to this project... thank you for your comment ☺

I've already added the entry for `es` in `src/message.ts`

File: es.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (antfu) [inline es.json:1]: This file should be in `locales`

**Comment by antfu** [comment]
Great, thanks

**Pull request merged**